docs/gettingstarted.md

```diff
@@ -173,6 +178,68 @@ procname were both passed in as command line parameters. If both of
 these parameters are passed in then default configuration settings will
 be used.
 
+The parameters following procname set the following:
+
+  |Parameter|                          Description|
+  |---------------------------------| ----------------------------|
+  |U|                                  Authentication requiring a usr:pwd file|
+  |localtime|                          Sets process running in local time rather than GMT|
+  |g|                                  Garbage collection immediate (1) or deferred (0)|
+  |T|                                  Timeout in seconds for client queries, 0 for no timeout|
+  |w|                                  Workspace MB limit|
+  |load|                               Files or database directory to load|
+  |startwithall|                       Determine if process is started when all is specified| 
+  |extras|                             Specify any additional parameters|
+  |qcmd|                               Allows different versions of q to be used or different command line options - rlwap, numactl|

```

Comment:
Line up this table correctly, so its easy to view for anyone editing this file in future.